src/simple_experiments/events/simple_card.cljs

```diff
@@ -1,15 +1,9 @@
 (ns simple-experiments.events.simple-card
-  (:require [re-frame.core :refer [reg-event-db reg-event-fx dispatch]]
-            [re-frame-fx.dispatch]
-            [cljs-time.core :as time]
-            [cljs-time.coerce :as timec]
-            [cljs-time.format :as timef]
-            [clojure.string :as string]
-            [simple-experiments.db :as db :refer [app-db]]
-            [simple-experiments.db.patient :as db-p]
-            [simple-experiments.events.utils :as u :refer [assoc-into-db]]))
-
-(defn six-digit-id [card-uuid]
+  (:require [clojure.string :as string]
+            [re-frame.core :refer [reg-event-fx]]
+            [simple-experiments.events.navigation :as nav]))
+
+(defn ->six-digit-id [card-uuid]
```

Comment:
See "Conversions and coercions" in https://stuartsierra.com/2016/01/09/how-to-name-clojure-functions. I also prefer the function name without the arrow.

Reply:
agreed generally, but that would prevent us from using local symbols called `six-digit-id` in this case.

Reply:
also, what's up with these:
```clojure
(defn dob->dob-string [dob]
  (timef/unparse
   (timef/formatter "dd-MMM-YYYY")
   (timec/from-long dob)))

(defn dob-string->time [dob-string]
  (some-> (timef/formatter "dd/MM/YYYY")
          (timef/parse dob-string)))

(defn age->dob-string [age]
  (-> (time/now)
      (time/minus (time/years age))
      (time/minus (time/days (rand-int 300)))
      dob->dob-string))

(defn dob-string->age [dob-string]
  (some-> dob-string
          dob-string->time
          (time/interval (time/now))
          time/in-years))
```

Reply:
maybe i should rename it to `uuid->six-digit-id`

Reply:
The time coercions are many different functions, and take very similar arguments. Hence the -> names. I had simple names first, and then transitioned into these when they got complex.

Renaming it for avoiding shadowing makes sense though.